### DISCUSSION TITLE
Readonly view

### DISCUSSION THREAD

**Discussion by Radiotechniman** [mealie-recipes/mealie#2165] [Feature Request]
Having a centralized DB of recepies is great and Mealie looks awesome, but in order to function (atleast in my household) the recepies need to be viewable by anyone without the need to have yet another login.

Hence this feature request, it is similar to the "household under groups" but I do not want to be setting up any access (nor groups)

How would it work? 
Personally I'd use the Ubooquity approach with a seperate URL for the display view..
And under "Site Settings"> Config there could be a flag which would enable this functionality 
There could be an option as well which woud restrict the IP addess(es) that can view the page, alternatively this page could run on a different port which would negate the need for any ip restrictions as this can then be sorted through the router.

**Comment by michael-genson** [comment]
This is one of the bigger features being worked on, though with no ETA yet:
https://github.com/hay-kot/mealie/issues/1326

And while not directly related, the idea of "households" as a unit under "groups" is also being considered:
https://github.com/hay-kot/mealie/discussions/2134

**Reply by Radiotechniman** [reply]
Thanks Michael, the public recepies of #1326 seems to cover my request
